src/utils/format-numbers.js

```diff
@@ -167,16 +167,35 @@ export function ratioToCssPercent(ratio: number): string {
   return (ratio * 100).toFixed(4) + '%';
 }
 
-export function formatBytes(bytes: number): string {
```

Comment:
Without this change, 1GB was displayed as 1.00GB (because it was using 3 significant digits). So I changed this so that we can take this value as a parameter.